src/TodosContext.tsx

```diff
@@ -0,0 +1,275 @@
+import React, {
+  createContext,
+  useCallback,
+  useContext,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import * as todoService from './api/todos';
+import { TodoContextHandlers, TodosContextValue } from './types/ContextValues';
+import { Errors } from './constants/Errors';
+import { FilterStatus } from './types/FilterStatus';
+import { useErrorMessage } from './components/Error/ErrorMessageContext';
+
+/* eslint-disable */
+export const TodosContext = createContext<
+  (TodosContextValue & TodoContextHandlers) | null
```

Comment:
Usually, we create some object with default empty values, like:

```
{
  todos: [],
  setTodos: () => {},
}
```
BTW, empty function `() => {}` might be stored in variable and reused in such cases